doc/Contributing.md

```diff
@@ -0,0 +1,52 @@
+# Introduction #
+
+The program is not finished yet, so we like your help. Current aim is to make the program "playable". To make this happen faster, we need developers.
+
+# Code development #
+
+Anything that does not work or exist needs fixing. You need to know C++, and make commits using the [code style](CodeStyle.md) of the project, and provide Doxygen comments with it.
+
+The code should build out of the box at a Linux system. For Windows, it is currently a little more difficult. See [BuildProgramVisualStudio](BuildProgramVisualStudio.md) for more details.
+
+If you need graphics that do not exist yet, just make some stub graphics. It does not matter how ugly they are.
+
+Submissions take the form of pull requests via Github.
+
+
+# Graphics development #
+
+We need LOTS of graphics. Most of the graphics are created using the [Blender program](http://www.blender.org). There are textures and templates in our repository that you should use, so everything gets a uniform look and a standardized format so it can be automagically processed without much effort.
+
+Note that the GPL v2 license of the project requires you to make all your source files publicly available, so others can build on your work. Some source files may not allow such publication, please check the license conditions beforehand.
+
+## RCD files ##
+For future contributions with graphics and item properties, an alternative route exists (in theory so far, nobody has tried it yet). On startup, the program reads RCD files to load its graphics and item properties. The point of this is that everybody can create such a file, and thus make his own extensions.
+
+Currently the **rcdgen** converter program exists (as in, you can download its source, and build it). There is also documentation what you can express in it, and the source files of the used graphics from the project data are available.
```

Comment:
Hmm, technically, it's documentation of the RCD format, not the input format of rcdgen. The data fields mostly overlap, but the rcdgen input has more (eg section names, mechanisms for adding lots of images, nice names for values, etc)
Not sure what to do here. It should be documented, I guess, perhaps as part of the output file spec?